part-spec-proto/ic_logic.json

```diff
@@ -86,50 +54,27 @@
             "title": "clock",
             "type": "object",
             "required": [
-                "partType",
-                "manufacturer",
-                "mpn",
-                "datasheetVersion"
+                "componentID"
```

Comment:
well since clocks may not have a fixed frequency, I was going to leave this one as optional. Feel free to change it if you want to further sub-divide this part though.